### ISSUE TITLE
Compilation error in tools/

### ISSUE THREAD

**Issue by amadanmath** [kaldi-asr/kaldi#3062]
I am getting this during `make` in `tools/openfst-1.6.7` (on Mac OS Mojave, 10.14 (18A391))

```
 cd . && /usr/local/bin/automake-1.16 --foreign Makefile
configure.ac: error: no proper invocation of AM_INIT_AUTOMAKE was found.
configure.ac: You should verify that configure.ac invokes AM_INIT_AUTOMAKE,
configure.ac: that aclocal.m4 is present in the top-level directory,
configure.ac: and that aclocal.m4 was recently regenerated (using aclocal)
```

I do not know much about automake, but I thought all the listed conditions were satisfied: there is `AM_INIT_AUTOMAKE([foreign nostdinc -Wall -Werror subdir-objects])` in `configure.ac`, there is `aclocal.m4`, and I even tried to run `aclocal` manually to regenerate it.

**Comment by danpovey** [comment]
It looks to me like you know more about automake than I do, so you would be
better placed to debug it.

I tried on my mac, also running mojave, and the installation seems to work
fine under both automake 1.15 and (after I upgraded it) automake 1.16.  I
was using brew to install this:

mac:kaldi: ls -l $(which automake)
lrwxr-xr-x  1 danielpovey  admin  40 Feb 28 13:32 /usr/local/bin/automake
-> ../Cellar/automake/1.16.1_1/bin/automake

Are you sure you don't have mixed versions installed somehow?

Dan
